apps/masterbots.ai/components/routes/chat/chat.tsx

```
@@ -33,278 +34,263 @@ export function Chat({
       onFinish(message) {
         saveNewMessage({
-          role: 'assistant',
+          role: "assistant",
           threadId:
             params.threadId || isNewChat ? threadId : activeThread?.threadId,
           content: message.content,
-          jwt: session!.user?.hasuraJwt
-        })
-      }
-    })
+          jwt: session!.user?.hasuraJwt,
```

Biome

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)